projects/components/facet/bootstrap/facet-filters/facet-filters.ts

```diff
@@ -130,4 +99,68 @@ export class BsFacetFilters implements OnChanges {
         return this.facetService.hasData(facet.aggregation, this.results);
     }
 
+    private addFacetMenu() {
+        let outFacets: Action[] = [];
+
+        outFacets.push(new Action({
+            name: `add_remove_all`,
+            text: this.userFacets.length < this.facets.length ? "Add all" : "Remove all",
+            icon: this.hasFacetSelected ? 
+                    (this.userFacets.length < this.facets.length ? "far fa-minus-square mr-1" : "far fa-check-square mr-1")
+                    : "far fa-square mr-1",
+            title: this.userFacets.length < this.facets.length ? "Add all facets" : "Remove all facets",
+            action: () => {
+                if (this.hasFacetSelected && this.userFacets.length === this.facets.length) this.facetService.removeAllFacet();
+                else this.facetService.addAllFacet();
+                this.buildFilters();
+            }
+        }));
+
+        for (let facet of this.facets) {
+            outFacets.push(new Action({
+                name: `add_${facet.name}`,
+                text: facet.title,
+                icon: facet.icon,
+                selected: !!this.userFacets?.find(userFacet => userFacet.name === facet.name),
+                title: `Add ${facet.title}`,
+                action: () => {
+                    if (this.userFacets?.find(userFacet => userFacet.name === facet.name)) this.facetService.removeFacet({name: facet.name, position: 0, hidden: false, expanded: true, view: ""})
+                    else this.facetService.addFacet({name: facet.name, position: 0, hidden: false, expanded: true, view: ""});
+                    this.buildFilters();
+                }
+            }));
+        }
+
+        let add_action = new Action({
+            name: "facets_config",
+            icon: "fas fa-cog",
+            title: "Facets customization",
+            children: outFacets
+        });
+        this.filters = [add_action, ...this.filters]; 
+    }
+
+    get filteredFacets() {
```

Comment:
I think that in the current implementation, enableCustomization=false will not work unless you already have facets stored in the user settings, so it might break existing customers. I think the solution is to completely ignore the user settings when enableCustomization=false (just display this.facets directly).

Reply:
You're right, I overlooked that sorry. I'll add a line in the `get filteredFacets()` to return `this.facets` in case `enableCustomization` is undefined or false.